src/$262Object.cpp

```diff
@@ -47,6 +48,19 @@ JS_DEFINE_NATIVE_FUNCTION($262Object::create_realm)
     return JS::Value(realm->$262());
 }
 
+JS_DEFINE_NATIVE_FUNCTION($262Object::detach_array_buffer)
+{
+    auto array_buffer = vm.argument(0);
+    VERIFY(array_buffer.is_object() && is<JS::ArrayBuffer>(array_buffer.as_object()));
```

Comment:
Hmm, perhaps we should handle this gracefully and throw. It's a wrong test then, but probably not worth a crash.

Reply:
sounds reasonable, fixed.